src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <div className="App">
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+        {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+        {isMarried === false
+          ? 'I am not married'
+          : `${partnerName} is my ${partner}`
+        }
+
+      </section>
```

Comment:
```suggestion
        <p className="Person__partner">
          {isMarried === false
            ? 'I am not married'
            : `${partnerName} is my ${partner}`
          }
        </p>
      </section>
```
That's the reason of failing tests
![image](https://user-images.githubusercontent.com/96409563/213990207-fecd7f97-b37f-4ba8-8bda-a3ad200d4beb.png)

Reply:
Thank you. Already fixed